br/pkg/lightning/backend/external/writer.go

```diff
@@ -384,8 +384,16 @@ func (w *Writer) flushKVs(ctx context.Context, fromClose bool) (err error) {
 			zap.Any("rate", float64(savedBytes)/1024.0/1024.0/time.Since(ts).Seconds()))
 	}()
 
-	slices.SortFunc(w.writeBatch[:], func(i, j common.KvPair) int {
-		return bytes.Compare(i.Key, j.Key)
+	// TODO: set maxGoroutines according to the available CPU count.
+	sorty.MaxGor = 8
```

Comment:
we already encoding/writing using configured concurrency, so each writer should restrict by it too.